bundle.md

```diff
@@ -16,7 +16,7 @@ See [`config.json`](config.md) for more details.
 
 2. <a name="containerFormat02" />A directory representing the root filesystem of the container.
 While the name of this REQUIRED directory may be arbitrary, users should consider using a conventional name, such as `rootfs`.
-This directory MUST be referenced from within the `config.json` file.
+This directory MUST be referenced by [root](config.md#root) within the `config.json` file.
```

Comment:
I think we want backticks around `root` to match [our][1] [existing][2] [pattern][3].

[1]: https://github.com/opencontainers/runtime-spec/blame/f56fe8ba78f9d70c63fbd301cf1cd680218742fa/config.md#L48
[2]: https://github.com/opencontainers/runtime-spec/blame/f56fe8ba78f9d70c63fbd301cf1cd680218742fa/config.md#L304
[3]: https://github.com/opencontainers/runtime-spec/blame/f56fe8ba78f9d70c63fbd301cf1cd680218742fa/config.md#L308